src/app.js

```diff
@@ -1,12 +1,82 @@
-import axios from 'axios';
+import axios from "axios";
 
-// Async - await Full data
-async function fetchData() {
+const countryList = document.getElementById('country-list');
+// const countryImage = document.getElementById('country-flag')
+
+//1. data fetchen
+
+console.log("test tekst")
+
+// const usernameInputField = document.getElementById('input-search');
+
+async function fetchDataCountries() {
+    const URI = 'https://restcountries.com/v2/all'
+    // const ENDPOINT = 'name'
     try {
-        const result = await axios.get('https://restcountries.com/v2/all');
-        console.log(result)
-    } catch(error) {
-        console.error(error);
+        const response = await axios.get(URI)
+        console.log(response.data);
+
+        response.data.sort ((a, b) => (a.population - b.population));
+
+
+            response.data.map((country) => {
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
            response.data.map(({ population, flag, region, name }) => {
```